lib/usd/translators/mayaReferenceUpdater.cpp

```diff
@@ -205,30 +205,48 @@ UsdMayaPrimUpdater::PushCopySpecs PxrUsdTranslators_MayaReferenceUpdater::pushCo
     (*dstEditRouter)(routerContext, routingData);
 
     // Retrieve the destination layer and prim path from the routing data.
-    auto dstLayerStr = findValue(routingData, TfToken("layer"));
```

Comment:
Renamed completely confusing "dst" prefixed variables.  The destination, passed in to this method as arguments, is the authored USD stage and its edit target layer.  The USD cache is a separate destination layer that gets referenced or payloaded in.